### PR TITLE
Boosted Creature

### PR DESCRIPTION
**Pull request by oualid6496**
To Do:
- [x] Roll Boosted Monster
- [x] Boosted monster shouldn't change if restart happens on the same day.
- [x] Boosted monster should change if restart happens on another day.
- [x] Boosted monster should get double exp.
- [x] Boosted monster should get double loot.
- [ ] Boosted monster should get double respawn.
- [x] There should be a output json file (to work with the client).
- [ ] On login there should be a message displaying which monster is boosted today.

Info:
- Path for the Json file needs to be configured in the lib file.
- List of possible boosted creatures can also be configured in the lib file (the list of bestiary is being used now)

### CODE REVIEW

**Comment by oualid6496** [comment]
if anyone wants to finish this feel free to do so :)

**Comment by majestyotbr** [comment]
@gpedro PR waiting for your review.

File: data/lib/others/boostedCreature.lua [hunk 1]
```diff
@@ -0,0 +1,672 @@
+-- Made by Nekiro
+if not BoostedCreature then
+	BoostedCreature = {name = ""}
+end
+
+BoostedCreature.jsonPath = "D:/xampp/htdocs/boostedCreature.json"
```
Comment (gpedro) [inline data/lib/others/boostedCreature.lua:6]: this should be fetched by database

File: data/lib/others/boostedCreature.lua [hunk 2]
```diff
@@ -0,0 +1,672 @@
...
+BoostedCreature.storages = {race = 99999, timestamp = 99998} -- make sure they arent the same and change to whatever you like
+
+BoostedCreature.races = {
```
Comment (gpedro) [inline data/lib/others/boostedCreature.lua:9]: can we reuse bestiary races?

**Comment by majestyotbr** [comment]
@marcosvf132 did a new PR for Boosted Creature. This can be closed.

**Pull request closed without merging**